collections/_article/rendering-priorities-january-2024.md

```diff
@@ -0,0 +1,116 @@
+
+**2. Background pipeline compilation (in progress)**
+
+This remains a high priority and has only been pushed back due to other foundational work on the RenderingDevice backend.
+
+**3. Shader compilation groups ([GH-79606](https://github.com/godotengine/godot/pull/79606)) (done)**
+
+We completed this and it was included in the 4.2 release! As a reminder, this optimization decreases shader compilation time for most 3D materials and should result in faster startup times an less stuttering in many games. 
+
+### Stability
+
+**1. BUGS (in progress, always)**
+
+We fixed over [200](https://github.com/godotengine/godot/pulls?q=is%3Apr+milestone%3A4.2+is%3Aclosed+label%3Atopic%3Arendering%2Ctopic%3Ashaders%2Ctopic%3Aparticles+label%3Abug) rendering related bugs in 4.2! 
+
+**2. Proper multi-threading in the RenderingDevice (in progress)**
+
+This was delayed due to foundational work on the RenderingDevice backend. But the work is currently underway and is showing promising [results](https://x.com/dariosamo/status/1735363059202375819?s=20) already!
+
+### Usability
+
+**1. GL Compatibility renderer - 3D (in progress)**
+
+We added shadows to the compatibility renderer in time for 4.2, as well as many other smaller features and improvements. We have already merged support for MSAA, lightmaps, and 3D scaling for 4.3 and will continue adding the remaining features. we aim to support Glow, ReflectionProbes, and post process adjustments as soon as possible.
+
+**2. FSR 2.2 / TAA improvements ([GH-81197](https://github.com/godotengine/godot/pull/81197)) (done)**
+
+We merged FSR2.2 during the 4.2 dev cycle as well as a host of other TAA enhancements. We have kept our current TAA as it is has less of a performance impact and can be useful when you aren't using upscaling. Our goal is to continue improving the performance and appearance of our TAA solution to offer a cheaper alternative to FSR2.2.
+
+## Others
+
+4.2 also brought a lot of exciting changes. Among them:
+
+**1. Overhauled ParticlesProcessMaterial ([GH-79527](https://github.com/godotengine/godot/pull/79527))**
+
+**2. TextureRD ([GH-79288](https://github.com/godotengine/godot/pull/79288)) (official demo)**
+
+**3. Significant LightmapGI enhancements to both performance and quality**
+
+**4. Vertex compression optimization ([GH-81138](https://github.com/godotengine/godot/pull/81138))**
+
+**5. 2D HDR to support using 3D glow in 2D ([GH-80215](https://github.com/godotengine/godot/pull/80215))**
+
+**6. Optional ANGLE backend to support low end devices ([GH-72831](https://github.com/godotengine/godot/pull/72831))**
+
+For more information, check out the "Rendering, particles, and shaders" section of the [4.2 release blog post](https://godotengine.org/article/godot-4-2-arrives-in-style/#rendering-particles-and-shaders).
+
+
+## Priorities
```

Comment:
Maybe should be "New priorities" or "Current priorities" to better make clearer this is separate to the previous Update section?

Reply:
```suggestion
## Current Priorities
```
Great idea